core/src/banking_stage/consumer.rs

```diff
@@ -760,6 +760,7 @@ impl Consumer {
         let fee_payer = message.fee_payer();
         let fee_budget_limits = FeeBudgetLimits::from(process_compute_budget_instructions(
             message.program_instructions_iter(),
+            &bank.feature_set,
```

Comment:
Split out #3922 - we can use cached value here.

Will wait on merge/review of that until this PR is merged.